lib/teracy-dev/plugin.rb

```diff
@@ -45,5 +58,33 @@ def self.sync(plugins)
     def self.installed?(plugin_name)
       return Vagrant.has_plugin?(plugin_name)
     end
+
+    def self.validate(plugin, config)
```

Comment:
don't introduce this type of complexity, as I can see it just checks the existence. So use Util.exist? instead